GVFS/GVFS.Platform.Mac/MacServiceProcess.cs

```diff
@@ -0,0 +1,77 @@
+using GVFS.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.Platform.Mac
+{
+    /// <summary>
+    /// Class to query the configured services on macOS
+    /// </summary>
+    public class MacServiceProcess

```

Comment:
I would name this something like `MacDaemonManager` because:

1. "service" is the Windows term.. the *NIX term is "daemon".
2. `MacServiceProcess` to me says this class models the Mac daemon process, but it actually wraps interactions with `launchctl`